BackofficeBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -106,13 +98,15 @@ public function testNewNodePageHome()
         $formUser['node[nodeSource]'] = 'root';
         $formUser['node[routePattern]'] = '/page-test' .time();
 
-        $crawler = $this->client->submit($formUser);
+        $this->client->submit($formUser);
         $crawler = $this->client->request('GET', '/admin/');
 
         $this->assertEquals($nbLink + 2, $crawler->filter('a')->count());
 
-        $crawler = $this->client->request('GET', '/api/node/' . $nodeName);
+        $this->client->request('GET', '/api/node/' . $nodeName);
         $this->assertEquals(200, $this->client->getResponse()->getStatusCode());
         $this->assertSame('application/json', $this->client->getResponse()->headers->get('content-type'));
+
+        $this->client->request('DELETE', '/api/node/' . $nodeName . '/delete');
```

Comment:
Could you check that the response is ok (quickly with the status code for exemple)